e2e/tests-dfx/generate.bash

```diff
@@ -113,3 +113,13 @@ teardown() {
     assert_file_not_exists "src/declarations/hello_backend/index.js"
     assert_file_not_exists "src/declarations/hello_backend/index.d.ts"
 }
+
+@test "dfx generate succeeds with an encrypted identity without input" {
+    dfx_start
+    dfx canister create --all
+
+    assert_command "${BATS_TEST_DIRNAME}/../assets/expect_scripts/init_alice_with_pw.exp"
+    assert_command dfx identity use alice
+    
+    assert_command timeout 30s dfx generate
```

Comment:
If you're not checking the output of the commands (assert_eq, assert_match and so forth), but only checking for success/failure, there is no need for `assert_command`:
```suggestion
    "${BATS_TEST_DIRNAME}/../assets/expect_scripts/init_alice_with_pw.exp"
    dfx identity use alice
    
    timeout 30s dfx generate
```

Reply:
I put assert_command in front of any command that the test could be considered to be testing, because I've run into several issues where I couldn't tell what command was failing because bats was eating the output. assert_command's output is much better.